docs/madr/2-sqlite.md

```diff
@@ -0,0 +1,98 @@
+# Using aos-sqlite to manage data registered in the ANT Registry Process
```

Comment:
The total should propose the problem, not necessarily the implementation decision

```suggestion
# Storing ANT Registry State in AO Process
```